lib/utils/isStandardSyntaxUrl.js

```diff
@@ -34,18 +31,14 @@ module.exports = function(url /*: string*/) /*: boolean*/ {
 	// Less variable works only at the beginning
 	// Check is less variable, allow use '@url/some/path'
 	// https://github.com/less/less.js/blob/3.x/lib/less/parser/parser.js#L547
-	if (url[0] === '@' && /^@@?[\w-]+$/.test(url)) {
+	if (url.startsWith('@') && /^@@?[\w-]+$/.test(url)) {
 		return false;
 	}
 
 	// In url without quotes scss variable can be everywhere
 	// But in this case it is allowed to use only specific characters
 	// Also forbidden "/" at the end of url
-	if (
-		url.indexOf('$') !== -1 &&
-		/^[$\sA-Za-z0-9+-/*_'"/]+$/.test(url) &&
-		url[url.length - 1] !== '/'
-	) {
+	if (url.includes('$') && /^[$\sA-Za-z0-9+-/*_'"/]+$/.test(url) && !url.endsWith('/')) {
```

Comment:
Actually I will revert the includes change; this should be made in the other branch